setup.py

```diff
@@ -22,6 +22,7 @@
     'Reductions': ['fairlearn==0.4.6'],
     'FairAdapt':['rpy2'],
     'notebooks': ['jupyter', 'tqdm', 'igraph[plotting]', 'lightgbm', 'seaborn', 'ipympl'],
+    'tempeh-datasets': ['tempeh'],
```

Comment:
can we call this `LawSchoolGPA` instead of `tempeh-datasets`?

Reply:
Fixed in latest